inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/pooling.hpp

```diff
@@ -48,6 +48,27 @@ typedef std::tuple<
         std::string                     // Device name
 > globalPoolLayerTestParamsSet;
 
+typedef std::tuple<
+        std::vector<size_t>,            // Kernel size
+        std::vector<size_t>,            // Stride
+        std::vector<size_t>,            // Dilation
+        std::vector<size_t>,            // Pad begin
+        std::vector<size_t>,            // Pad end
+        ngraph::op::RoundingType,       // Rounding type
+        ngraph::op::PadType             // Pad type
+> maxPoolV8SpecificParams;
+
+typedef std::tuple<
+        maxPoolV8SpecificParams,
+        InferenceEngine::Precision,     // Net precision
+        InferenceEngine::Precision,     // Input precision
+        InferenceEngine::Precision,     // Output precision
+        InferenceEngine::Layout,        // Input layout
+        InferenceEngine::Layout,        // Output layout
+        std::vector<size_t>,            // Input shape
```

Comment:
InputShape?

Reply:
As per our road map, we should implement dynamic shapes tests in shared part if it does not lead to big changes in other plugins and repositories, otherwise just in CPU specific part. Here you are creating new tests and can avoid further refactoring.